### PR TITLE
[Bach/MH] Version v0.0.21

### PR DESCRIPTION
**Pull request by jansentom**

### CODE REVIEW

File: bach/docs/source/example-notebooks/explore-data.rst [hunk 1]
```diff
@@ -37,9 +37,10 @@ We first have to instantiate the model hub and an Objectiv DataFrame object.
 .. doctest:: explore-data
 	:skipif: engine is None
 
-	>>> # instantiate the model hub, and set the default time aggregation to daily
+	>>> # instantiate the model hub, set the default time aggregation to daily
+	>>> # and get the application & path global contexts
 	>>> from modelhub import ModelHub
```
Comment (hendrik-obj) [inline bach/docs/source/example-notebooks/explore-data.rst:42]: also 
```suggestion
	>>> from modelhub import ModelHub
	>>> from bach import display_sql_as_markdown
```

File: bach/docs/source/example-notebooks/open-taxonomy.rst [hunk 2]
```diff
@@ -8,182 +8,603 @@
...
+.. testsetup:: open-taxonomy
+	:skipif: engine is None
+
+	start_date = '2022-06-01'
+	end_date = '2022-06-30'
+
+.. doctest:: open-taxonomy
+	:skipif: engine is None
+
+	>>> from modelhub import ModelHub
```
Comment (hendrik-obj) [inline bach/docs/source/example-notebooks/open-taxonomy.rst:60]:
```suggestion
	>>> from modelhub import ModelHub
	>>> from bach import display_sql_as_markdown
```

File: bach/docs/source/example-notebooks/user-intent.rst [hunk 3]
```diff
@@ -27,15 +27,27 @@ Besides the open model hub, we have to import the following packages for this ex
     import pandas as pd
```
Comment (hendrik-obj) [inline bach/docs/source/example-notebooks/user-intent.rst:27]: add 
`from modelhub import ModelHub`

**Review by hendrik-obj — COMMENTED** [review]
some imports were still missing in the example pages. other than that all good

File: bach/docs/source/example-notebooks/explore-data.rst (hunk 1 above)
Comment (ivarpruijn) [reply bach/docs/source/example-notebooks/explore-data.rst:42]: Adding `from bach import display_sql_as_markdown` breaks the doctests actually. Let's not do this now, and I'll fix it later.

**Pull request merged**